### PR TITLE
Unclear instruction

### PR DESCRIPTION
**Pull request by needfulthing**
Select “New InputEventAction” and click the new “InputEvent”.

There is no "InputEvent" to click on. The action tab already shows up after selecting “New InputEventAction”.

### CODE REVIEW

**Comment by Calinou** [comment]
Thanks!

**Comment by Calinou** [comment]
I think the current documentation was almost there. It should have said:

> click the new "InputEventAction"

instead of:

> click the new "InputEvent"

I guess the one who wrote this part of the documentation didn't notice the name was trimmed due to the inspector being narrow.

**Comment by needfulthing** [comment]
I forgot to mention this. Yes, it's a confusion because the narrowed field shows only "InputEvent" in the image, the "Action" part is hidden.

File: getting_started/step_by_step/your_first_game.rst [hunk 1]
```diff
@@ -1225,9 +1225,9 @@ is using the "Shortcut" property of the ``Button`` node.
 
 In the ``HUD`` scene, select the ``StartButton`` and find its *Shortcut* property
 in the Inspector. Select "New Shortcut" and click on the "Shortcut" item. A
-second *Shortcut* property will appear. Select "New InputEventAction" and click
-the new "InputEvent". Finally, in the *Action* property, type the name ``ui_select``.
-This is the default input event associated with the spacebar.
+second *Shortcut* property will appear. Select "New InputEventAction". Finally,
+in the *Action* property, type the name ``ui_select``. This is the default input
+event associated with the spacebar.
```
Comment (mhilbrunner) [inline getting_started/step_by_step/your_first_game.rst:1230]:
```suggestion
second *Shortcut* property will appear. Select "New InputEventAction" and click
the new "InputEventAction". Finally, in the *Action* property, type the name ``ui_select``.
This is the default input event associated with the spacebar.
```

**Comment by mhilbrunner** [comment]
I've gone and applied Calinou's suggestion. Thanks for pointing this out, and congrats for your first docs contribution!

**Pull request merged**